pkg/kubelet/dockershim/docker_stats_windows.go

```diff
@@ -98,7 +86,10 @@ func (ds *dockerService) getContainerStats(containerID string) (*runtimeapi.Cont
 		WritableLayer: &runtimeapi.FilesystemUsage{
 			Timestamp: timestamp,
 			FsId:      &runtimeapi.FilesystemIdentifier{Mountpoint: info.DockerRootDir},
-			UsedBytes: &runtimeapi.UInt64Value{Value: uint64(*containerJSON.SizeRw)},
+			// used bytes from image are not implemented on Windows
+			// don't query for it since it is expensive to call docker over named pipe
+			// https://github.com/moby/moby/blob/1ba54a5fd0ba293db3bea46cd67604b593f2048b/daemon/images/image_windows.go#L11-L14
+			UsedBytes: &runtimeapi.UInt64Value{Value: 0},
```

Comment:
Do you have a sense of how much of the perf improvements you see on Windows are due to info from the docker api vs how much is due to not querying for this data on Windows?

Reply:
not this specifically but I added a comment here with some details https://github.com/kubernetes/kubernetes/pull/104287#issuecomment-964702227